Mathlib/LinearAlgebra/Finsupp.lean

```diff
@@ -1189,25 +1184,33 @@ theorem splittingOfFinsuppSurjective_splits (f : M →ₗ[R] α →₀ R) (s : S
   rw [sum_single_index, one_smul]
   · exact (s (Finsupp.single x 1)).choose_spec
   · rw [zero_smul]
-#align linear_map.splitting_of_finsupp_surjective_splits LinearMap.splittingOfFinsuppSurjective_splits
+#align
+  linear_map.splitting_of_finsupp_surjective_splits
+  LinearMap.splittingOfFinsuppSurjective_splits
```

Comment:
Please don't do this. The linter has been configured to allow long `#align` statements. And it makes them much more greppable if they are in fact on one line.